src/index.js

```diff
@@ -0,0 +1,29 @@
+import './style.css';
+
+const tasks = [
+  {
+    index: 0,
+    completed: false,
+    description: 'review this project',
+  },
+  {
+    index: 1,
+    completed: true,
+    description: 'aprobe this project',
+  },
+];
+
+const fillList = () => {
+  const container = document.querySelector('.todo-list');
+  tasks.forEach((todo) => {
+    container.innerHTML += `<li>
+    <div class="taskCont">
+      <input type="checkbox">
+      <div class="task">${todo.description}</div>
+    </div>
+    <div class="icon">&#8942</div>
+  </li>
+`;
+  });
+};
+fillList();
```

Comment:
**OPTIONAL** - You have done well so far, but to help make your project more readable, I suggest you call this function when the content in the DOM has loaded. i.e attach it to the event listener `DOMContentLoaded`. Please fix this 😄